sdcm/cluster.py

```diff
@@ -2748,7 +2748,9 @@ def _gen_cqlsh_cmd(self, command, keyspace, timeout, host, port, connect_timeout
                       auth_params=auth_params, use_keyspace=use_keyspace, timeout=timeout,
                       connect_timeout=connect_timeout, ssl_params=ssl_params)
 
-        if self.parent_cluster.connection_bundle_file:
+        if self.parent_cluster.connection_bundle_file and (
+                packaging.version.parse(self.scylla_version) >= packaging.version.parse('5.2.0')):
```

Comment:
```
>>> import packaging.version
>>> packaging.version.parse("5.2.0-dev-0.20221220.bbfb9556fc88") >= packaging.version.parse('5.2.0')
False
>>> packaging.version.LegacyVersion("5.2.0-dev-0.20221220.bbfb9556fc88") >= packaging.version.LegacyVersion('5.2.0')
False
>>> packaging.version.LegacyVersion("5.2.0-dev-0.20221220.bbfb9556fc88") >= packaging.version.LegacyVersion('5.2.0-dev')
True
>>> packaging.version.parse("5.2.0-dev-0.20221220.bbfb9556fc88") >= packaging.version.parse('5.2.0-dev')
False
```

Reply:
you are right, even that the version received here isn't the full version, but just `5.2.0~dev`